mozzi_config.h

```diff
@@ -91,5 +91,11 @@ You need to have \#define STEREO_HACK true in mozzi_config.h
 
 //#define EXTERNAL_AUDIO_OUTPUT true
 
+// ESP32 with PT8211 DAC, requires AUDIO_RATE 32768
+#define ESP32_PT8211 true
```

Comment:
Since this is specific to ESP32, please move this to AudioConfigESP32.h.

Also (nitpicking) consider using defines along the way laid out there (for ESP_AUDIO_OUT_MODE). Keep in mind that more than just these two options (internal DAC vs PT2811) may exist in the future. A true/false define does not scale well with that.

Reply:
Gotcha.